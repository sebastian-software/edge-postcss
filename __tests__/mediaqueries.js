import { compile } from "./core"

// ====================================================
// =============== MEDIA QUERIES ======================
// ====================================================

test("Media Query Min/Max", () =>
  compile("@media screen and (width >= 500px) and (width <= 1200px) { .elem { display: block; } }")
)

test("Custom Media", () =>
  compile(`
    @custom-media --small-viewport (max-width: 30em);
    @media (--small-viewport) {
      body { font-size: 12px; }
    }
  `)
)


