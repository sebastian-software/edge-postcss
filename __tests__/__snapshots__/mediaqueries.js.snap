// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Media 1`] = `
"@media (max-width:30em){
body{
font-size:12px
}
}"
`;

exports[`Media Query Min/Max 1`] = `
"@media screen and (min-width:500px) and (max-width:1200px){
.elem{
display:block
}
}"
`;
